### ISSUE TITLE
o.change with qmetro

### ISSUE THREAD

**Issue by ramagottfried** [CNMAT/CNMAT-odot#282]
I'm fairly sure this is a bug with o.change, but I guess it could also be o.route?
o.change incorrectly thinks the bundle has changed -- not sure why, I tried inserting a o.validate but it still happens.  (haven't reviewed the source yet)

<pre><code>
----------begin_max5_patcher----------
982.3oc6YssaaiCD8Y6uBA8rShFRpa6GPedeuMHPVh0Vcko7JQkjtE8ee4M4
X2XWyvnPjTX.CJqQCm4LCGNGJ6eLeV3x1Go8gA+UvmClM6GymMSIRJXl49Yg
aJdrronWoVHaXyRZW3B8i1VzUrgxoc2QYEKanRUhLOqcf2P47uukpseX3hfv
kErUgA2ZTom+c8bBGMnv70LwzTNC14Ed45Z1p65nkbswP.bczh.BRNFGIGQh
uuyxB6n8uxPHiz5JkyZW9sqvgRQ+b9b4vBKC9Mz99hUzQvxoOpfS3mFZZ96h
x+gxCRSBxEfCkkCnQ8ZpYzx1AlRYxISO1lVPmNsf0IDR9d4kjSmWfmmWPouQ
IlLRvUPTJNOljkE4+TCgPTUIPjyolj2zZFLFQRH9OwffWchI1kDyxANukEdp
Pax5TDqBuXTp9x3n8gG3zVBd6pUMTmZTVKW0mrHGCpHGQFGeAQdlKQNi9fXx
OqfWD9csAwSyBN57gcLVdAhNG4vwh67ILtautS3MZvMOTypZeH72QO9TSgIr
.fPRjWxie4rjnHWRDsWW1tYaa+ya5YxA2rlVuZMW1YSr7r3KrQ480+GUBckX
AViu8n8CQmue3WaY7uVTdvFMoLoGToMYifmPWYaSam1Hp5kmFfCyWunRvr38
5rhydQ8TgwHuu3dZ0cKGXUMz6ZnrU70RsxHG64UE7ByQAhEK4YB7CoxA8VA4
fTVDHhNK9fDJSTyOWNq3CrD.lGnzHhXrqVsH7nLHYzZDhYp5OIxaQ+djfQVC
.HdTCD5Hg3NmqzwR2eEFBtcJaETtVzkidxd.6s+2stAmunjjo5Fj3vYlQn2w
773DMAWL3LOO7wjm2D4DvYZdzDR28u9hm2D1ootSyiuvt4.6lIwCQ4NytAwW
X2tvtMkrais+0UitvtAIuiY2.7qkcK+iI4lIvcmbK8iH2lIpeEbaw+Q7Frl7
.jF47avdgh2AJ9w79qfhGmbgh+BE+TRwOxA5NEOhre2.07T6K+k+ALkukxOL
A02NzUNFnlecLQRdm6qn87ZVAuVbdf8TJUnDrSo00UUT19apqp6kjoJTFczk
Jawi7T0mGOIdCOHafSr+fCXCdNbM8MEOVU9f7GdrY8BR8FdrpbFR7GdRrAOf
+VuHVfmb+AGaplIdCNXaVs72lKrEnwesBSsDM9I2Xy9brG2maUePr+viM0Nf
+VtfLavi+Z77Kt5D6zcsur9ThEa2dOsq2XSETDGX9a5W4Jcg51Zl9VkEC6n2
WOpOVIonSbhVt33rCc5C.+Xl9cjB2zVQ6XC0pWkZtzy+b9+CEfHaW.
-----------end_max5_patcher-----------
</code></pre>

**Comment by maccallum** [comment]
I'm not sure what the bug is here. Looks fine to me---can you comment @ramagottfried ?

**Comment by ramagottfried** [comment]
hmm, yeah, this looks fixed now. 

A couple weeks ago while I was working on my svg demo (with the current version), I came across the same problem in a different context. I dug around in o.change to see what was going on, did some tests, and now have a feeling that some functions don't correctly fill in the \0's bytes in the #bundle format which gets exposed by o.change. 

Unfortunately I can't reproduce it now, and I made the assumption that this example I had posted above  was the same issue -- I'll keep an eye out for this other problem, and we can close this one.
